components/Layout/HomeHeader/HomeHeader.vue

```diff
@@ -44,6 +44,7 @@ export default Vue.extend({
         limit: this.pages,
         fields: [ItemFields.Overview],
         enableImageTypes: [ImageType.Backdrop, ImageType.Logo],
+        includeItemTypes: ['Episode', 'Movie', 'MusicAlbum', 'Series'],
```

Comment:
Why not use ``ItemType``? We use it for Image types, this would make things consistent imo.